api/src/main/java/org/openmrs/module/datafilter/impl/api/db/hibernate/InterceptorUtil.java

```diff
@@ -14,7 +14,6 @@
 import org.hibernate.SessionFactory;
 import org.openmrs.api.context.Context;
 
-
```

Comment:
Discard this change please

Reply:
This is not from me, but rather from the code formatter.  Is this not worth including?

Reply:
You should revert it then